app/src/main/java/com/codesoom/assignment/domain/TaskRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.codesoom.assignment.domain;
+
+
+import java.util.List;
+import java.util.Optional;
+
+
```

Comment:
불필요한 공백 라인을 삭제해 주세요.
```suggestion

import java.util.List;
import java.util.Optional;

```